Makefile

```diff
@@ -99,6 +99,7 @@ USE_MODULE_DB=true
 REGRESS = $(patsubst test/sql/%.sql,%,$(TESTS))
 REGRESS_OPTS = \
 	$(TEST_INSTANCE_OPTS) \
+	--create-role=super_user,regular_user,regular_user2 \
```

Comment:
Make these users into `TEST_XXX` variables above and export them  like `TEST_PGUSER` above so that they are visible to the `runner.sh` script. That script should specify the same variables, but allow them to be overridden if already specified (i.e., `TEST_PGSUPERUSER=${TEST_PGSUPERUSER:-super_user}`).